backend/reservations/management/commands/fix_room.py

```diff
@@ -7,14 +7,26 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('number',
                             help='The room number')
+        parser.add_argument('--hotel-name',
+                            default='Ballys',
+                            help='The hotel name. Defaults to Ballys.')
 
     def handle(self, *args, **kwargs):
         if 'number' not in kwargs:
             raise CommandError("Must specify room number")
 
         room = None
+        hotel = None
+        if kwargs['hotel_name'].lower() == 'ballys':
+            hotel = 'Ballys'
+        elif kwargs['hotel_name'].lower() == 'hard rock' or \
+           kwargs['hotel_name'].lower() == 'hardrock':
```

Comment:
nit: indent looks off by a space, maybe just put on same line as above?

Reply:
UGH, thanks emacs.

Reply:
still off by one, in the other direction. align with line below